pkg/shell/nav.jsx

```diff
@@ -196,7 +196,7 @@ export function CockpitNavItem(props) {
                 </a>
                 {s && s.type && <PageStatus status={s} name={props.name} />}
                 { props.actions &&
-                    <div className="event-eater" onClick={event_eater} onKeyPress={event_eater}>
+                    <div className="nav-host-action-buttons event-eater" onClick={event_eater} onKeyPress={event_eater}>
```

Comment:
this is gonna break stuff, as I compare if `class == 'event-eater'`. If this is really needed, then we also need to check just for sub-string.